pkg/webreg/webreg.go

```diff
@@ -248,6 +250,7 @@ const templateDefinitions = `
 
 {{ define "chainTable" }}
 	<h2 id="chains">Chains</h2>
+	<p>Chains are registry components that allow users to string together multiple steps under one name.</p>
```

Comment:
What about `to string together multiple references as its steps`?  It is not quite obvious that a `Reference` is the same as a `step` IMHO

Reply:
Yeah, this ref/reference/step makes understanding this all a bit harder for me too.